.env

```diff
@@ -1,6 +1,6 @@
 DATABASE_URL= 
-DEV_DATABASE_URL=postgres://postgres:phantom@127.0.0.1:5432/users
-TEST_DATABASE_URL=postgres://postgres:phantom@127.0.0.1:5432/test_db
+DEV_DATABASE_URL=postgres://postgres:ally@127.0.0.1:5432/development
+TEST_DATABASE_URL=postgres://postgres:ally@127.0.0.1:5432/test
 ACCESS_TOKEN_SECRET = swsh23hjddnnsij
 ACCESS_TOKEN_LIFE = 7days
 PORT = 3020
```

Comment:
Why did you push `.env` ?

Reply:
going to refresh, as its in .gitignore

Reply:
This is the commit that introduced it: https://github.com/atlp-rwanda/eagles-phantom-backend/commit/0f451a4b323516784be39bb256d6d8655423af05

You probably just rebased.